bigquery/snippets/snippet.go

```diff
@@ -405,6 +470,121 @@ func queryLegacy(client *bigquery.Client, sqlString string) error {
 	return nil
 }
 
+func queryLegacyLargeResults(client *bigquery.Client, dstDatasetID, dstTableID string) error {
+	ctx := context.Background()
+	// [START bigquery_query_legacy]
+	q := client.Query(
+		"SELECT corpus FROM [bigquery-public-data:samples.shakespeare] GROUP BY corpus;")
+	q.UseLegacySQL = true
+	q.AllowLargeResults = true
+	q.QueryConfig.Dst = client.Dataset(dstDatasetID).Table(dstTableID)
+	// [END bigquery_query_legacy]
+	return runAndRead(ctx, client, q)
+}
+
+func queryWithArrayParams(client *bigquery.Client) error {
+	ctx := context.Background()
+	// [START bigquery_query_params_arrays]
+	q := client.Query(
+		`SELECT name, sum(number) as count 
+        FROM ` + "`bigquery-public-data.usa_names.usa_1910_2013`" + `
+        WHERE gender = @gender
+        AND state IN UNNEST(@states)
+        GROUP BY name
+        ORDER BY count DESC
+		LIMIT 10;`)
```

Comment:
I find the zig-zag indentation in this query text distracting.

Reply:
went more idiomatic with the formatting to make the query more readable.  if you want more inlined style I can minimize the number of lines.